src/Components/Common/FilePreviewDialog.tsx

```diff
@@ -66,153 +56,107 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
-                                  disabled={button[3] as boolean}
-                                >
-                                  <i className={`fas fa-${button[1]} mr-2`} />
-                                  {button[0] as string}
-                                </button>
-                              ))}
-                            </>
-                          )}
-                        </div>
-                        <div className="flex gap-3">
-                          {downloadURL && downloadURL.length > 0 && (
-                            <a
-                              href={downloadURL}
-                              download={`${file_state.name}.${file_state.extension}`}
-                              className="z-50 rounded bg-white/60 px-4 py-2 text-black backdrop-blur transition hover:bg-white/70"
-                            >
-                              <i className="fas fa-download mr-2" />
-                              Download
-                            </a>
-                          )}
-                          <button
-                            onClick={onClose}
-                            className="z-50 rounded bg-white/60 px-4 py-2 text-black backdrop-blur transition hover:bg-white/70"
-                          >
-                            <i className="fas fa-times mr-2" />
-                            Close
-                          </button>
-                        </div>
-                      </div>
-                      <div className="h-[80vh] w-full">
-                        {file_state.isImage ? (
-                          <img
-                            src={fileUrl}
-                            alt="file"
-                            className={
-                              "mx-auto object-contain " +
-                              zoom_values[file_state.zoom]
-                            }
-                            style={{
-                              transform: `rotate(${file_state.rotation}deg)`,
-                            }}
-                          />
-                        ) : (
-                          <iframe
-                            title="Source Files"
-                            src={fileUrl}
-                            className="mx-auto h-5/6 w-5/6 border-2 border-black bg-white md:my-6 md:w-4/6"
-                          />
-                        )}
-                      </div>
-                    </div>
-                  ) : (
-                    <div className="flex h-screen items-center justify-center">
-                      <div className="text-center">
-                        <CircularProgress />
-                      </div>
-                    </div>
-                  )}
-                </Dialog.Panel>
-              </Transition.Child>
+                    <i className="fas fa-download mr-2" />
```

Comment:
You should be using CareIcons in newer implementations. We are phasing out usage of font-awesome icons (all solid icons especially)

Reply:
Sure.